Tests/SwiftStdlibTests/StringExtensionsTests.swift

```diff
@@ -77,7 +77,12 @@ final class StringExtensionsTests: XCTestCase {
     func testIsEmail() {
```

Comment:
@omaralbeik There are some valid/invalid email test cases [here](https://blogs.msdn.microsoft.com/testing123/2009/02/06/email-address-test-cases/) 
Maybe it's worth to consider adding some of them on the tests :))

Reply:
Great, I added tests from links as well 💯